redis/Makefile

```diff
@@ -0,0 +1,98 @@
+
+REDIS_VERSION:=3.0.7
```

Comment:
Ok, I see. Was/is there a copyright notice on the original file we should apply here, too?

More generally, introducing the make file doesn't make me super-happy: it adds another thing to know and be aware of, and it won't easily work on Windows (usage of curl etc.) I'm ok with adding it temporarily for now, but we should replace this with some Maven build steps subsequently. Looking at the performed steps, it shouldn't be too difficult to do.